multifile/paster/vcf/__main__.py

```diff
@@ -0,0 +1,24 @@
+import argparse
+
+from multifile.paster.vcf.paster import VCFPaster
+
+
+def main():
+    parser = argparse.ArgumentParser(description="Merge VCF horizontally.")
+    parser.add_argument('input_paths', type=str, nargs='+', help="paths to files to merge")
+    parser.add_argument('output_path', type=str, help="path to output file")
+    parser.add_argument('-d', '--delimeter', type=str, default='\t', help="delimeter")
+    parser.add_argument('-b', '--batch-size', type=int, default=999, help='size of each input batch to process one at a time')
+    parser.add_argument('--n-header-lines', type=int, default=11, help="number of header lines (will be read only once)")
+    parser.add_argument('--n-fixed-cols', type=int, default=9, help="number of fixed columns on the left")
+    parser.add_argument('-t', '--temp-dir-path', type=str, help="path to directory for storing temporary files while pasting")
```

Comment:
What's the default location? `/tmp`? I think we should respect `$TMPDIR` if the user has set it.

Reply:
If the `--temp-dir-path` is not specified, then the python implementation of `tempfile` will look for OS environments etc.. `$TMPDIR` is the first one it will try:
https://github.com/python/cpython/blob/master/Lib/tempfile.py#L150

Reply:
Maybe we should advise this in the help message. @giang-nghg

Reply:
Can something to that effect be put in the command line help? As a user, I don't want to have to read the code to work that out.

Reply:
Good idea, I'll put that in the help text.